
From: Christoph Hellwig <hch@lst.de>

Not exactly a thing we want done from modules, and no module uses it
anyway.

Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/kernel/exit.c |    2 --
 1 files changed, 2 deletions(-)

diff -puN kernel/exit.c~unexport-exit_mm kernel/exit.c
--- 25/kernel/exit.c~unexport-exit_mm	2004-09-26 17:20:38.372582160 -0700
+++ 25-akpm/kernel/exit.c	2004-09-26 17:20:38.376581552 -0700
@@ -515,8 +515,6 @@ void exit_mm(struct task_struct *tsk)
 	__exit_mm(tsk);
 }
 
-EXPORT_SYMBOL(exit_mm);
-
 static inline void choose_new_parent(task_t *p, task_t *reaper, task_t *child_reaper)
 {
 	/*
_
